pgpy/packet/fields.py

```diff
@@ -467,28 +481,85 @@ def parse(self, packet):
         self.y = MPI(packet)
 
 
+class ECPoint:
+    def __init__(self, packet=None):
+        if packet is None:
+            return
+        xy = bytearray(MPI(packet).to_mpibytes()[2:])
+        self.format = ECPointFormat(xy[0])
+        del xy[0]
+        if self.format == ECPointFormat.Standard:
+            xylen = len(xy)
+            assert xylen % 2 == 0
+            self.bytelen = xylen // 2
+            self.x = MPI(MPIs.bytes_to_int(xy[:self.bytelen]))
+            self.y = MPI(MPIs.bytes_to_int(xy[self.bytelen:]))
+        elif self.format == ECPointFormat.Native:
+            self.x = bytes(xy)
+            self.y = None
+        else:
+            raise NotImplemented("No curve is supposed to use only X or Y coordinates")
```

Comment:
`raise NotImplementedError(...)`

`NotImplemented` is intended to be a return value rather than an exception, to signal that a comparison function (e.g. `__lt__` hasn't been implemented in a way that doesn't break list sorts